astro_2.0/features/Discover/components/Tokens/Tokens.tsx

```diff
@@ -151,6 +154,16 @@ export const Tokens: FC = () => {
     }
   }, [activeView, query.dao, isMounted]);
 
+  const getUnit = () => {
```

Comment:
In theory, it would be a bit better to calculate this value in variable with useMemo.

Reply:
Thanks. Will be fixed in tech debt.